tests/test_interface.rb

```diff
@@ -1321,7 +1352,7 @@ def test_interface_ipv4_all_interfaces
       validate_interface_shutdown(inttype_h)
       validate_vrf(inttype_h)
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
```

Comment:
I tested against all platforms (except for 8k) and did not see `channel group` remaining after running `default interface ethernet 1/1`